### PR TITLE
metric(ticdc): update changefees status dashboard

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref  https://github.com/pingcap/tiflow/issues/1606
 
### What is changed and how it works?
update changefees status dashboard to display the status label.
image
image

#### Tests 
 - Manual test
 - No code

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -23799,7 +23830,7 @@
     ]
   },
   "timezone": "browser",
-  "title": "Test-Cluster-TiCDC",
+  "title": "${DS_TEST-CLUSTER}-TiCDC",
   "uid": "YiGL8hBZ1",
-  "version": 45
+  "version": 1
```
Comment (overvenus) [inline metrics/grafana/ticdc.json:23835]: Why are these changed?

Comment (hi-rustin) [reply metrics/grafana/ticdc.json]: reverted.

**Review by overvenus — APPROVED** [review]
